cmd/neutron_query_relayer/metrics/metrics.go

```diff
@@ -111,13 +121,25 @@ func RecordActionDuration(action string, dur float64) {
 }
 
 func IncSuccessTxSubmit() {
-	relayerRequests.With(prometheus.Labels{
+	submittedTxCounter.With(prometheus.Labels{
 		labelType: typeSuccess,
 	}).Inc()
 }
 
 func IncFailedTxSubmit() {
-	relayerProofs.With(prometheus.Labels{
+	submittedTxCounter.With(prometheus.Labels{
 		labelType: typeFailed,
 	}).Inc()
 }
+
+func SetSubscriberTaskQueueNumElements(numElements int) {
+	subscriberTaskQueueNumElements.With(prometheus.Labels{}).Set(float64(numElements))
+}
+
+func IncQueriesToProcess() {
+	queriesToProcess.With(prometheus.Labels{}).Inc()
```

Comment:
why not a gauge with `.Set(subscriber.activeQueries)`? will simplify the `if !queryExists` part in the `processUpdateEvent`

Reply:
In tracking task it is required to use a counter. If more people from team think it is better to use gauge here — I will switch.

Reply:
agree with @sotnikov-s 
since @foxpy is unavailable now, i can fix it